lib/modules/hosts/photobucket.js

```diff
@@ -5,22 +5,23 @@ export default {
 	name: 'photobucket',
 	domains: ['photobucket.com'],
 	logo: '//pic2.pbsrc.com/common/favicon.ico',
-	detect: href => (/([is]?)[0-9]+|media|smg|img(?=.photobucket.com)/i).exec(href),
-	async handleLink(elem, [, prefix]) {
-		const href = elem.href.replace('.html', '');
+	detect: ({ href }) => (/([is]?)[0-9]+|media|smg|img(?=.photobucket.com)/i).exec(href),
+	async handleLink(href, [, prefix]) {
+		let src = href.replace('.html', '');
 
 		// user linked direct image so no need to hit API
-		if (prefix === 'i') {
-			elem.src = href;
-		} else {
+		if (prefix !== 'i') {
 			const { imageUrl } = await ajax({
 				url: 'https://api.photobucket.com/v2/media/fromurl',
 				data: { url: href },
```

Comment:
`const href` was changed to `let src` above; this needs to be changed too.
![image](https://cloud.githubusercontent.com/assets/7673145/15615142/90892e7e-240a-11e6-87e1-7d1cb1560e71.png)